pulseapi/entries/views.py

```diff
@@ -21,8 +21,8 @@ def post_validate(request):
     nonce = False
 
     if request.data:
-        csrf_token = request.data['csrfmiddlewaretoken']
-        nonce = request.data['nonce']
+        csrf_token = request.data.get('csrfmiddlewaretoken', False)
```

Comment:
yeah, it's a property of Python's dictionaries. You can either do direct access with array notation, or you can use `get` with a fallback, https://docs.python.org/2/library/stdtypes.html#dict.get